db-init.sh

```diff
@@ -9,4 +9,9 @@ aws dynamodb create-table --cli-input-json file://db-json/create-voter-table.jso
 aws dynamodb batch-write-item --request-items file://db-json/populate-election-table.json --endpoint-url http://localhost:8000  --output json --no-paginate
 
 aws dynamodb batch-write-item --request-items file://db-json/populate-voter-table-1.json --endpoint-url http://localhost:8000  --output json --no-paginate
-aws dynamodb batch-write-item --request-items file://db-json/populate-voter-table-2.json --endpoint-url http://localhost:8000  --output json --no-paginate
\ No newline at end of file
+aws dynamodb batch-write-item --request-items file://db-json/populate-voter-table-2.json --endpoint-url http://localhost:8000  --output json --no-paginate
+
+aws dynamodb delete-table --table-name abc_voter_sessions_local --endpoint-url http://localhost:8000  --output json --no-paginate
+aws dynamodb create-table --cli-input-json file://db-json/create-voter-sessions-table.json --endpoint-url http://localhost:8000  --output json --no-paginate
+
+aws dynamodb scan --table-name abc_voter_sessions_local --endpoint-url http://localhost:8000  --output json --no-paginate
```

Comment:
can you document the command somewhere (here or README or docs-init.sh or something) to sync the docs files to the s3 bucket?